weekly-openmeteo/README.md

```diff
@@ -2,7 +2,87 @@
 This folder contains the code for the [OpenMeteo API](https://open-meteo.com/en/docs) data pipeline. This includes `extract.py`, `transform.py`, `load.py` and `pipeline.py`. This pipeline uses the `requests` library to send HTML GET requests to the `OpenMeteo API` and the `psycopg2` library to load this data onto our database. The data requested includes `time of sunset` and `time of sunrise`. This pipeline is therefore designed to run every week as these are far more predictable values.
 
 ## Setup
+1. Ensure that an SQL server RDS has been setup prior and is accessible
+    - Note: be sure to store the credentials for accessing this database safely.
+
+### To Run Locally:
+2. Setup a venv and install the requirements
+```bash
+python -m venv venv
+source venv/bin/activate
+pip install -r requirements.txt
+```
+3. Create a `.env` file with the following:
+    - DB_HOST - to access your RDS instance
+    - DB_USER - username for accessing RDS
+    - DB_PASSWORD - password for accessing RDS
+    - DB_NAME - the name of your database
+    - DB_PORT - port to access RDS (typically 5432 for postgres)
```

Comment:
Nice to explain what each env variable should contain, even with descriptive names